app/controllers/support/base_controller.rb

```diff
@@ -1,13 +1,24 @@
 class Support::BaseController < ApplicationController
+  include Pundit
+
   before_action :require_dfe_user!
+  after_action :verify_authorized
+  after_action :verify_policy_scoped, only: %i[index results] # # rubocop:disable Rails/LexicallyScopedActionFilter
```

Comment:
`index` makes sense, but it seems awkward to be defining actions with `only` on a non-standard method (results) that's only in a couple of controllers - TBH I can see what Rubocop is getting at here.

In fact, having checked the code in question, arguably the right thing to do here is rename the `results` methods to `index`, but even with or without that, there's a Rubocop-compliant alternative:

```
after_action :verify_policy_scoped, if: :index_method?

...
private
  def index_method?
    %i[index results].include?(action_name)
  end
  
```

Reply:
Thanks for the suggestion, I've adopted it. Renaming `results` to `index` didn't feel quite right since the `results` action is current a POST in both instances, not a GET, and making it into a GET with a query param would have meant putting the emails search strings into the URL...